src/Libraries/Microsoft.Extensions.AI/ChatCompletion/ChatClientBuilderServiceCollectionExtensions.cs

```diff
@@ -13,59 +13,67 @@ public static class ChatClientBuilderServiceCollectionExtensions
     /// <summary>Registers a singleton <see cref="IChatClient"/> in the <see cref="IServiceCollection"/>.</summary>
     /// <param name="serviceCollection">The <see cref="IServiceCollection"/> to which the client should be added.</param>
     /// <param name="innerClient">The inner <see cref="IChatClient"/> that represents the underlying backend.</param>
+    /// <param name="lifetime">The service lifetime for the client.</param>
     /// <returns>A <see cref="ChatClientBuilder"/> that can be used to build a pipeline around the inner client.</returns>
     /// <remarks>The client is registered as a singleton service.</remarks>
     public static ChatClientBuilder AddChatClient(
         this IServiceCollection serviceCollection,
-        IChatClient innerClient)
-        => AddChatClient(serviceCollection, _ => innerClient);
+        IChatClient innerClient,
+        ServiceLifetime lifetime)
```

Comment:
Note that overloads accepting implementation instances also accept lifetimes, since the `ChatClientBuilder` pattern admits application of transformer methods post-registration.